advanced_functionality/r_kernel/using_r_with_amazon_sagemaker.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>Typo in Translate</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Fixed